tp-test/play.go

```diff
@@ -266,22 +267,66 @@ func runTest(ctx context.Context, round TestRound, db1 *sql.DB, db2 *sql.DB) err
 	return nil
 }
 
-func doStmt(ctx context.Context, conn sqlz.ConnContext, round string, stmt sqlgen.Stmt) (*sqlz.ResultSet, error) {
-	// FIXME: prepared statement
+func doStmt(ctx context.Context, conn sqlz.ConnContext, round string, stmt sqlgen.Stmt, db *sql.DB) (rs *sqlz.ResultSet, err error) {
+	stmtDone := make(chan struct{})
+	dumpDone := make(chan struct{})
+	buf := new(strings.Builder)
+	go func() {
+		defer close(dumpDone)
+		needDump := false
+		select {
+		case <-stmtDone:
+			if err != nil && strings.Contains(strings.ToLower(err.Error()), "lock") {
+				needDump = true
+			}
+		case <-time.After(3 * time.Second):
```

Comment:
Do we need to make it a timeout parameter?

Reply:
I add a command flag, the code smells a little bad...